gateway/server.go

```diff
@@ -303,6 +306,11 @@ func home(w http.ResponseWriter, r *http.Request) {
 	// And need to re route the end user to an end of session
 	switch head {
 	case "get_server_models":
+		base.Zlog.Infof("GET SERVER MODEL")
+		if base.ValidateClientIP(r) == false {
+			http.Error(w, "Service is not available", 401)
```

Comment:
@susmithp I think we need a more easily identifiable or distinguishable log message here i.e. via Zlog for easy retrieval of all forbidden domain / ip blocks and then also some of the other more broader error handling issues discussed by @vejmarie

Reply:
@arunkoshy I added that line just for debugging, and have removed it now.